docs/index.md

```diff
@@ -7,3 +7,4 @@
 * [Truffle language interoperability and Sulong](INTEROP.md)
 * [Sulong-related publications](PUBLICATIONS.md)
 * [Sulong's test cases](TESTS.md)
+* [How to get Sulong installed](INSTALLATION.md)
```

Comment:
Can we change that to `[Distribution-specific installation of the external dependencies](DEPENDENCIES.md)` or so? I think this file name and description match its content better.